.travis.yml

```diff
@@ -0,0 +1,108 @@
+# Travis CI (MIT License) configuration file for the Twenty Sixteen theme.
+# @link https://travis-ci.org/
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Declare versions of PHP to use. Use one decimal max.
+php:
+  # aliased to a recent 5.5.x version
+  - "5.5"
+  # aliased to a recent 5.4.x version
+  - "5.4"
+  # aliased to a recent 5.3.x version
+  - "5.3"
+  # Current $required_php_version for WordPress: 5.2.4
+  - "5.2"
+
+# Declare which versions of WordPress to test against.
+# Also declare whether or not to test in Multisite.
+env:
+  # Trunk
+  # @link https://github.com/WordPress/WordPress
+  - WP_VERSION=master WP_MULTISITE=0
+  - WP_VERSION=master WP_MULTISITE=1
+  # WordPress 4.3
+  # @link https://github.com/WordPress/WordPress/tree/4.3-branch
+  - WP_VERSION=4.3 WP_MULTISITE=0
+  - WP_VERSION=4.3 WP_MULTISITE=1
```

Comment:
> The default theme should have a minimum version of the version it is released on

Minimum required version for Twenty Sixteen is 4.2.

I doubt this testing matrix even matters since all we're doing is running the theme through Coding Standards, right now at least. Happy to make any adjustments after discussion during this week's meeting.